Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3061,6 +3059,16 @@ static FloatRect computeMarqueeSelectionRect(const WebCore::FloatPoint& point1,
         return { scrollDeltaWidth, scrollDeltaHeight };
     }();
 
+#if PLATFORM(MAC)
+    IntSize scrollAdjustmentBasedOnScreenBoundaries;
```

Comment:
this and the `scrollDelta += scrollAdjustmentBasedOnScreenBoundaries` part can also be inside the `if (RefPtr page = this->page())` condition, and that way the `scrollAdjustmentBasedOnScreenBoundaries` variable can also become immutable